slatedb/src/merge_iterator.rs

```diff
@@ -256,8 +263,8 @@ mod tests {
         let mut iters: VecDeque<TestIterator> = VecDeque::new();
         iters.push_back(
             TestIterator::new()
-                .with_entry(b"aaaa", b"0000", 5)
-                .with_entry(b"aaaa", b"1111", 6)
+                .with_entry(b"aaaa", b"0000", 6)
```

Comment:
This change is because the old way violates the way seqnums appear in our ssts, right?